crates/oci/Cargo.toml

```diff
@@ -22,10 +23,17 @@ reqwest = "0.12"
 serde = { workspace = true }
 serde_json = { workspace = true }
 spin-common = { path = "../common" }
+spin-componentize = { path = "../componentize" }
+spin-compose = { path = "../compose" }
```

Comment:
I apologise for harping on this but it breaks my heart that the OCI crate has to know about this stuff (but I understand that you've looked into this and it's the least worst way)

Reply:
I managed to eliminate the `spin-componentize` dependency by unifying the loader from trigger and here into a `ComponentSourceLoaderFs` that lives in `spin-compose`.